python/lsst/sims/maf/metricBundles/metricBundleGroup.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, bundleDict, dbObj, outDir='.', resultsDb=None, verbose=True,
         self.bundleDict = bundleDict
         # Check the dbObj.
         if not isinstance(dbObj, db.Database):
-            raise ValueError('dbObj should be an instantiated lsst.sims.maf.db.Database (or child) object.')
+            warnings.warn('Warning: dbObj should be an instantiated lsst.sims.maf.db.Database (or child) object.')
```

Comment:
Why change it to a warning? Did you want to be able to use metricBundles without a dbObj?